apps/examples/testcase/ta_tc/wifi_manager/itc/itc_wifi_manager_main.c

```diff
@@ -805,33 +1082,47 @@ int wifi_manager_itc(int argc, FAR char *argv[])
 	itc_wifi_manager_set_mode_p();
 
 	itc_wifi_manager_get_mode_p(); // set softap mode inside this function
-
 	itc_wifi_manager_connect_disconnect_ap_p(); // change to station mode and try to connect to ap
 
 	itc_wifi_manager_scan_ap_p(); // Reinitialized wifi manager with the callback hander for scan results
 
+	WIFITEST_WAIT;
+
 	itc_wifi_manager_reinit_p();
 
 	itc_wifi_manager_redeinit_p();
 
 	itc_wifi_manager_redisconnect_p();
 
 	itc_wifi_manager_connect_long_ssid_p();
-
 	itc_wifi_manager_average_joining_ap(); // Disable DoS preventation from Access Point
 
 	itc_wifi_manager_average_leaving_ap(); // Disable DoS preventation from Access Point
 
 	itc_wifi_manager_average_stoping_ap(); // Disable DoS preventation from Access Point
 
 	itc_wifi_manager_success_ratio_ap();
```

Comment:
Please check that the success ratio works correctly. The output messages are broken.